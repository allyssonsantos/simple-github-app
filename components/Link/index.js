import styled from 'styled-components';

const Link = styled.a`
  color: inherit;
  text-decoration: none;
`;

export default Link;
