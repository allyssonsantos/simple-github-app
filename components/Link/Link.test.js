import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Link from './';

describe('With Snapshot Testing', () => {
  it('Should match snapshot', () => {
    const component = renderer.create(<Link>This is a link</Link>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
