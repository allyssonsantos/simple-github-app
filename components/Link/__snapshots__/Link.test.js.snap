// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Should match snapshot 1`] = `
.c0 {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

<a
  className="c0"
>
  This is a link
</a>
`;
